src/cloud/social/provider.ts

```diff
@@ -44,6 +47,8 @@ interface Invite {
   key: string;
   // True iff uProxy has root access on the server, i.e. uProxy deployed it.
   isAdmin?: boolean;
+  // (optional) host key that should be used to verify the server
```

Comment:
Two things:
 - no need to say `(optional)` because the type tells us that
 - let's give the exact format of this field - it looks like it should be base64-encoded, is that right? (also: is it right to say, this is the same value you'd see in your `known_hosts` file?)